feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go

```diff
@@ -143,9 +144,17 @@ func verifyNodeTelemetry(t *testing.T, node, peer gnmi.DeviceOrOpts, nodePort, p
 				return true
 			}
 			gotLen = len(intf.Neighbor)
-			return gotLen == 0
+			if *deviations.LLDPInterfaceConfigOverrideGlobal {
```

Comment:
if block ends with a return statement, so drop this else and outdent its block

https://google.github.io/styleguide/go/decisions#indent-error-flow

Reply:
Done